### PR TITLE
Change worfkow engine name in README.md files

### PR DESCRIPTION
**Pull request by mhamra**
# Description
Related issue: #5458 

This PR changes the name of the workflow engine in the README.md files.

### CODE REVIEW

File: deployability/README.MD [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Deployability general documentation
+## JobFlow - Deployability general documentation
```
Comment (fcaffieri) [inline deployability/README.MD:1]: JobFlow is a module within deployability, we should remove it from here

File: deployability/modules/allocation/README.MD [hunk 2]
```diff
@@ -1,12 +1,12 @@
-## Allocation Module
+## JobFlow - Allocation Module
```
Comment (fcaffieri) [inline deployability/modules/allocation/README.MD:1]: JobFlow is a module within deployability just like Allocation, we should remove it from here

File: deployability/modules/provision/README.MD [hunk 3]
```diff
@@ -1,14 +1,14 @@
-## Provision module
+## JObFlow - Provision module
```
Comment (fcaffieri) [inline deployability/modules/provision/README.MD:1]: JobFlow is a module within deployability just like Provision, we should remove it from here

File: deployability/modules/testing/README.MD [hunk 4]
```diff
@@ -1,4 +1,4 @@
-## Testing Module
+## JobFlow - Testing Module
```
Comment (fcaffieri) [inline deployability/modules/testing/README.MD:1]: JobFlow is a module within deployability just like Testing, we should remove it from here

**Review by fcaffieri — CHANGES_REQUESTED** [review]
Good Job, only there are still some references within:

- readme.md: for example there are references to workflow_engine without modifying, an example:
https://github.com/wazuh/wazuh-qa/blob/enhancement/5458-change-workflow-name-in-readme/deployability/README.MD?plain=1#L44-L51

- References in setup.py:
https://github.com/wazuh/wazuh-qa/blob/enhancement/5458-change-workflow-name-in-readme/deployability/modules/setup.py#L28-L29
https://github.com/wazuh/wazuh-qa/blob/enhancement/5458-change-workflow-name-in-readme/deployability/modules/setup.py#L32
https://github.com/wazuh/wazuh-qa/blob/enhancement/5458-change-workflow-name-in-readme/deployability/modules/setup.py#L39-L41

- Examples of executions, an example:
https://github.com/wazuh/wazuh-qa/blob/enhancement/5458-change-workflow-name-in-readme/deployability/modules/workflow_engine/examples/agent/aws/test-agent-basic-info.yaml#L2C19-L2C27

File: deployability/modules/provision/README.MD (hunk 3 above)
Comment (rauldpm) [inline deployability/modules/provision/README.MD:1]: Is `JobFlow` not `JObFlow`

**Review by rauldpm — CHANGES_REQUESTED** [review]

File: deployability/README.MD (hunk 1 above)
Comment (rauldpm) [reply deployability/README.MD:1]: The developed workflow tool (initially called `Workflow engine` (orchestrator) is a composition of multiple modules), has been renamed to `JobFlow`

These modules can work without the orchestrator but they are part of the JobFlow package

File: deployability/modules/allocation/README.MD (hunk 2 above)
Comment (rauldpm) [reply deployability/modules/allocation/README.MD:1]: The developed workflow tool (initially called `Workflow engine` (orchestrator) is a composition of multiple modules), has been renamed to `JobFlow`

These modules can work without the orchestrator but they are part of the JobFlow package

File: deployability/modules/provision/README.MD (hunk 3 above)
Comment (rauldpm) [reply deployability/modules/provision/README.MD:1]: The developed workflow tool (initially called `Workflow engine` (orchestrator) is a composition of multiple modules), has been renamed to `JobFlow`

These modules can work without the orchestrator but they are part of the JobFlow package

File: deployability/modules/testing/README.MD (hunk 4 above)
Comment (rauldpm) [reply deployability/modules/testing/README.MD:1]: The developed workflow tool (initially called `Workflow engine` (orchestrator) is a composition of multiple modules), has been renamed to `JobFlow`

These modules can work without the orchestrator but they are part of the JobFlow package

File: deployability/modules/provision/README.MD (hunk 3 above)
Comment (rauldpm) [reply deployability/modules/provision/README.MD:1]: This should be removed in the Federico request changes

File: deployability/modules/jobflow/tests/conftest.py [hunk 5]
```diff
@@ -19,7 +19,7 @@
 @pytest.fixture
 def logger_mock(request) -> MagicMock:
     """Fixture to mock common logger methods."""
-    logger_to_patch = request.param.get('logger_to_patch', "workflow_engine.workflow_processor.logger")
+    logger_to_patch = request.param.get('logger_to_patch', "jobflow.workflow_processor.logger")
```
Comment (pro-akim) [inline deployability/modules/jobflow/tests/conftest.py:22]: Shouldn't the name of the workflow_processor method be changed?

File: deployability/modules/jobflow/tests/conftest.py [hunk 6]
```diff
@@ -39,9 +39,9 @@ def dag(request) -> DAG:
         gl_dag = graphlib.TopologicalSorter()
         dep_dict = {'task1': 'task2'}
         with patch.object(gl_dag, 'prepare'), \
-            patch('workflow_engine.workflow_processor.DAG._DAG__build_dag',
+            patch('jobflow.workflow_processor.DAG._DAG__build_dag',
                 return_value=(gl_dag, dep_dict)), \
-            patch('workflow_engine.workflow_processor.DAG._DAG__create_execution_plan',
+            patch('jobflow.workflow_processor.DAG._DAG__create_execution_plan',
```
Comment (pro-akim) [inline deployability/modules/jobflow/tests/conftest.py:44]: Shouldn't the name of the workflow_processor method be changed?

File: deployability/modules/jobflow/tests/conftest.py [hunk 7]
```diff
@@ -62,11 +62,11 @@ def w_processor(request) -> WorkflowProcessor:
     threads = request.param.get('threads', 1)
     log_level = request.param.get('log_level', 'info')
     schema_file = request.param.get('schema_file', 'schema.yaml')
-    with patch("workflow_engine.workflow_processor.WorkflowFile") as file_mock:
+    with patch("jobflow.workflow_processor.WorkflowFile") as file_mock:
```
Comment (pro-akim) [inline deployability/modules/jobflow/tests/conftest.py:65]: Shouldn't the name of the workflow_processor method and WorkflowFilw be changed?

File: deployability/modules/jobflow/tests/test_dag.py [hunk 8]
```diff
@@ -6,12 +6,12 @@
 from unittest.mock import patch, MagicMock, call
 import pytest
 
-from workflow_engine.workflow_processor import DAG
+from jobflow.workflow_processor import DAG
```
Comment (pro-akim) [inline deployability/modules/jobflow/tests/test_dag.py:9]: Shouldn't the name of the workflow_processor method and WorkflowFilw be changed?

File: deployability/modules/jobflow/tests/test_workflow_file.py [hunk 9]
```diff
@@ -6,17 +6,17 @@
 from unittest.mock import patch, MagicMock, call, mock_open
 import pytest
 
-from workflow_engine.workflow_processor import WorkflowFile
+from jobflow.workflow_processor import WorkflowFile
```
Comment (pro-akim) [inline deployability/modules/jobflow/tests/test_workflow_file.py:9]: Shouldn't the name of the workflow_processor method and WorkflowFile class name be changed?

File: deployability/modules/jobflow/tests/test_workflow_processor.py [hunk 10]
```diff
@@ -8,17 +8,17 @@
 from unittest.mock import patch, MagicMock, call
 import pytest
 
-from workflow_engine.workflow_processor import WorkflowProcessor, DAG
-from workflow_engine.task import ProcessTask, TASKS_HANDLERS
+from jobflow.workflow_processor import WorkflowProcessor, DAG
```
Comment (pro-akim) [inline deployability/modules/jobflow/tests/test_workflow_processor.py:11]: Shouldn't the name of the WorkflowProcessor class be changed?

File: deployability/README.MD (hunk 1 above)
Comment (mhamra) [reply deployability/README.MD]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/8b07c4235ad62ada42e8e8e0eae015a7a465538b

File: deployability/modules/allocation/README.MD (hunk 2 above)
Comment (mhamra) [reply deployability/modules/allocation/README.MD]: Adjusted in a new commitModified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/8b07c4235ad62ada42e8e8e0eae015a7a465538b

File: deployability/modules/provision/README.MD (hunk 3 above)
Comment (mhamra) [reply deployability/modules/provision/README.MD]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/8b07c4235ad62ada42e8e8e0eae015a7a465538b

File: deployability/modules/testing/README.MD (hunk 4 above)
Comment (mhamra) [reply deployability/modules/testing/README.MD]: Removed in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/8b07c4235ad62ada42e8e8e0eae015a7a465538b

File: deployability/modules/jobflow.egg-info/PKG-INFO [hunk 11]
```diff
@@ -0,0 +1,8 @@
+Metadata-Version: 2.1
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/modules/jobflow.egg-info/PKG-INFO]: Removed in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/d44439711871e1c97d7d4ef1df5f3e5797c8818e

File: deployability/modules/jobflow.egg-info/SOURCES.txt [hunk 12]
```diff
@@ -0,0 +1,73 @@
+setup.py
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/modules/jobflow.egg-info/SOURCES.txt]: Removed in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/d44439711871e1c97d7d4ef1df5f3e5797c8818e

File: deployability/modules/jobflow.egg-info/dependency_links.txt [hunk 13]
```diff
@@ -0,0 +1 @@
+
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/modules/jobflow.egg-info/dependency_links.txt]: Removed in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/d44439711871e1c97d7d4ef1df5f3e5797c8818e

File: deployability/modules/jobflow.egg-info/entry_points.txt [hunk 14]
```diff
@@ -0,0 +1,2 @@
+[console_scripts]
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/modules/jobflow.egg-info/entry_points.txt]: Removed in commit https://github.com/wazuh/wazuh-qa/commit/d44439711871e1c97d7d4ef1df5f3e5797c8818e

File: deployability/modules/jobflow.egg-info/not-zip-safe [hunk 15]
```diff
@@ -0,0 +1 @@
+
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/modules/jobflow.egg-info/not-zip-safe]: Removed in commit https://github.com/wazuh/wazuh-qa/commit/d44439711871e1c97d7d4ef1df5f3e5797c8818e

File: deployability/modules/jobflow.egg-info/top_level.txt [hunk 16]
```diff
@@ -0,0 +1 @@
+jobflow
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/modules/jobflow.egg-info/top_level.txt]: Removed in commit https://github.com/wazuh/wazuh-qa/commit/d44439711871e1c97d7d4ef1df5f3e5797c8818e

File: deployability/README.MD [hunk 17]
```diff
@@ -35,20 +35,20 @@ git checkout {project-branch}
 pip3 install -r deployability/deps/requirements.txt
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/README.MD:35]: Updated in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/5df00d792e0c9b030c82fec1f9ad16a6951d4584

File: deployability/modules/jobflow/tests/conftest.py [hunk 18]
```diff
@@ -7,7 +7,7 @@
 from unittest.mock import patch, MagicMock
 import pytest
 
-from workflow_engine.workflow_processor import DAG, WorkflowProcessor
+from jobflow.workflow_processor import DAG, WorkflowProcessor
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/modules/jobflow/tests/conftest.py]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8

File: deployability/modules/jobflow/tests/conftest.py (hunk 5 above)
Comment (mhamra) [reply deployability/modules/jobflow/tests/conftest.py]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8

File: deployability/modules/jobflow/tests/conftest.py [hunk 19]
```diff
@@ -39,9 +39,9 @@ def dag(request) -> DAG:
         gl_dag = graphlib.TopologicalSorter()
         dep_dict = {'task1': 'task2'}
         with patch.object(gl_dag, 'prepare'), \
-            patch('workflow_engine.workflow_processor.DAG._DAG__build_dag',
+            patch('jobflow.workflow_processor.DAG._DAG__build_dag',
```
[the remark this answers is not in the document]
Comment (mhamra) [reply deployability/modules/jobflow/tests/conftest.py]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8

File: deployability/modules/jobflow/tests/conftest.py (hunk 7 above)
Comment (mhamra) [reply deployability/modules/jobflow/tests/conftest.py]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8

File: deployability/modules/jobflow/tests/test_dag.py (hunk 8 above)
Comment (mhamra) [reply deployability/modules/jobflow/tests/test_dag.py]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8

File: deployability/modules/jobflow/tests/test_workflow_file.py (hunk 9 above)
Comment (mhamra) [reply deployability/modules/jobflow/tests/test_workflow_file.py]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8

File: deployability/modules/jobflow/tests/test_workflow_processor.py (hunk 10 above)
Comment (mhamra) [reply deployability/modules/jobflow/tests/test_workflow_processor.py]: Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8

**Review by pro-akim — APPROVED** [review]
## Review Notes
LGTM

File: deployability/README.MD (hunk 17 above)
[the remark this answers is not in the document]
Comment (rauldpm) [reply deployability/README.MD:35]: image

It is `JobFlow`, not `Jobflow`

File: deployability/README.MD [hunk 20]
```diff
@@ -35,27 +35,27 @@ git checkout {project-branch}
...
-> Note:  It is possible to find some fixture examples in deployability/modules/workflow_engine/examples/
+> Note:  It is possible to find some fixture examples in deployability/modules/jobflow/examples/
 
 
 Example:
 
 ```
 version: 0.1
-description: This workflow is used to test agents' deployment for DDT1 PoC
+description: This jobflow is used to test agents' deployment for DDT1 PoC
```
Comment (rauldpm) [inline deployability/README.MD:58]: > This jobflow is used

This YAML is used

File: deployability/README.MD [hunk 21]
```diff
@@ -263,11 +263,11 @@ python3 -m workflow_engine modules/workflow_engine/examples/dtt1-agents-poc.yaml
 
 - Modules
 
-The framework has 4 modules (`allocation`, `provision`, `testing`, `observability`) that must act consecutively, and a 5th module that orchestrates the previous modules (`Workflow`).
+The framework has 4 modules (`allocation`, `provision`, `testing`, `observability`) that must act consecutively, and a 5th module that orchestrates the previous modules (`Jobflow`).
```
Comment (rauldpm) [inline deployability/README.MD:266]: JobFlow

File: deployability/README.MD [hunk 22]
```diff
@@ -263,11 +263,11 @@ python3 -m workflow_engine modules/workflow_engine/examples/dtt1-agents-poc.yaml
...
 
 |Module|Description
 |--|--|
-|Workflow (Orchestrator)|Receives a YAML containing all the instructions to execute for the test development. It contains instructions for allocation, provision, and testing.
+|Jobflow (Orchestrator)|Receives a YAML containing all the instructions to execute for the test development. It contains instructions for allocation, provision, and testing.
```
Comment (rauldpm) [inline deployability/README.MD:270]: JobFlow

File: deployability/README.MD [hunk 23]
```diff
@@ -299,18 +299,18 @@ Deployability contains the following directories:
 
 #### Overview
 
-![image](https://github.com/wazuh/wazuh-qa/assets/125690423/d09dfa61-67cb-410d-8677-bd904d5c4d51)
+![image](https://github.com/wazuh/wazuh-qa/assets/2949519/f1a11353-d61f-4afe-9290-b3c33f619f60)
```
Comment (rauldpm) [inline deployability/README.MD:302]: JobFlow

File: deployability/README.MD [hunk 24]
```diff
@@ -299,18 +299,18 @@ Deployability contains the following directories:
...
 
 #### Allocator, provision & test process
 
-![image](https://github.com/wazuh/wazuh-qa/assets/125690423/b1e24a43-13d7-4035-a3e5-5ace4667ec10)
+![image](https://github.com/wazuh/wazuh-qa/assets/2949519/ca927c80-dcde-4b6d-a72e-9c5b6ed601df)
```
Comment (rauldpm) [inline deployability/README.MD:306]: JobFlow

File: deployability/README.MD [hunk 25]
```diff
@@ -299,18 +299,18 @@ Deployability contains the following directories:
...
 
 
 
 ----
 
 
-[draw-plot.zip](https://github.com/wazuh/wazuh-qa/files/14330528/plot.zip)
+[draw-plot.zip](https://github.com/user-attachments/files/15778407/draw-plot.zip)
```
Comment (rauldpm) [inline deployability/README.MD:313]: JobFlow

File: deployability/modules/allocation/README.MD [hunk 26]
```diff
@@ -46,30 +46,30 @@ To use this module, you should use a Debian-based system, we recommend using Ubu
...
 
-      >Note: It is possible to find some fixture examples in [deployability/modules/workflow_engine/examples/](../workflow_engine/examples)
+      >Note: It is possible to find some fixture examples in [deployability/modules/jobflow/examples/](../jobflow/examples)
 
       Example:
 
       ```bash
       version: 0.1
-      description: This workflow is used to test agents' deployment for DDT1 PoC
+      description: This jobflow is used to test agents' deployment for DDT1 PoC
```
Comment (rauldpm) [inline deployability/modules/allocation/README.MD:72]: This YAML

File: deployability/modules/allocation/README.MD [hunk 27]
```diff
@@ -350,7 +350,7 @@ python3 modules/allocation/main.py --action create --provider vagrant --size lar
 
 #### General-specific functions
 
-- **Launcher** ([/wazuh-qa/deployability/modules/allocation/main.py](main.py)): The entry point for the workflow or the user who wishes to execute a test.
+- **Launcher** ([/wazuh-qa/deployability/modules/allocation/main.py](main.py)): The entry point for the jobflow or the user who wishes to execute a test.
```
Comment (rauldpm) [inline deployability/modules/allocation/README.MD:353]: I don't think jobflow is the correct word there

File: deployability/modules/allocation/README.MD [hunk 28]
```diff
@@ -373,10 +373,10 @@ python3 modules/allocation/main.py --action create --provider vagrant --size lar
 
 #### Diagram
 
-![image](https://github.com/wazuh/wazuh-qa/assets/64099752/d302f96b-70c0-4fbc-869b-914973ee8186)
+![image](https://github.com/wazuh/wazuh-qa/assets/2949519/e6721533-e84a-4563-8746-f30645fd5367)
```
Comment (rauldpm) [inline deployability/modules/allocation/README.MD:376]: JobFlow

File: deployability/modules/allocation/README.MD [hunk 29]
```diff
@@ -373,10 +373,10 @@ python3 modules/allocation/main.py --action create --provider vagrant --size lar
...
 
 
-[Allocation.drawio.zip](https://github.com/wazuh/wazuh-qa/files/14868775/Allocation.drawio.zip)
+[Allocation.drawio.zip](https://github.com/user-attachments/files/15788198/Allocation.drawio.zip)
```
Comment (rauldpm) [inline deployability/modules/allocation/README.MD:379]: JobFlow

File: deployability/modules/jobflow/README.MD [hunk 30]
```diff
@@ -1,8 +1,8 @@
-## Workflow engine
+## JobFlow  engine
 
 ### User documentation
 
-The execution of the Workflow is done through the installation of its library.
+The execution of the Jobflow is done through the installation of its library.
```
Comment (rauldpm) [inline deployability/modules/jobflow/README.MD:5]: JobFlow

File: deployability/modules/jobflow/README.MD [hunk 31]
```diff
@@ -29,26 +29,26 @@ To use this module, you should use a Debian-based system, we recommend using Ubu
 	pip3 install -r deployability/deps/requirements.txt
 	```
 
-4. Install the Workflow engine library and its launcher:
+4. Install the JobFlow  engine library and its launcher:
```
Comment (rauldpm) [inline deployability/modules/jobflow/README.MD:32]: Double space between JobFlow and engine

File: deployability/modules/jobflow/README.MD [hunk 32]
```diff
@@ -29,26 +29,26 @@ To use this module, you should use a Debian-based system, we recommend using Ubu
...
 
-      >Note: It is possible to find some fixture examples in deployability/modules/workflow_engine/examples/
+      >Note: It is possible to find some fixture examples in deployability/modules/jobflow/examples/
 
       Example:
 
       ```bash
       version: 0.1
-      description: This workflow is used to test agents' deployment for DDT1 PoC
+      description: This jobflow is used to test agents' deployment for DDT1 PoC
```
Comment (rauldpm) [inline deployability/modules/jobflow/README.MD:51]: This YAML

File: deployability/modules/jobflow/README.MD [hunk 33]
```diff
@@ -255,12 +255,12 @@ To use this module, you should use a Debian-based system, we recommend using Ubu
 
 ### Technical documentation
 
-`Workflow Engine` is the orchestrator of the deployability test architecture.
+`JobFlow  engine` is the orchestrator of the deployability test architecture.
```
Comment (rauldpm) [inline deployability/modules/jobflow/README.MD:258]: Double space

File: deployability/modules/jobflow/README.MD [hunk 34]
```diff
@@ -255,12 +255,12 @@ To use this module, you should use a Debian-based system, we recommend using Ubu
...
 
 Its function is to allow the ordered and structured execution in steps of allocation, provision, and testing.
 
-`The Workflow Engine` receives instructions through a `YAML document`, the structure of which can be exemplified in tests found in:
-`wazuh-qa/deployability/modules/workflow_engine/examples`
+`The JobFlow  engine` receives instructions through a `YAML document`, the structure of which can be exemplified in tests found in:
```
Comment (rauldpm) [inline deployability/modules/jobflow/README.MD:262]: Double space

**Review by rauldpm — CHANGES_REQUESTED** [review]
I am not going to review the rest of it because it is too repetitive

- The name of the module is JobFlow, not Jobflow or jobflow
- If you are referring to the module directory, a path, or the pip module, it is jobflow because it is coded like that
- Do not use mass replace, there are a lot of double spaces between words
- What happened to the .gitignore file? There is a workflow reference 
- Some sentences should be reviewed as jobflow does not match the context

File: deployability/README.MD (hunk 22 above)
Comment (mhamra) [reply deployability/README.MD]: Fixed
https://github.com/wazuh/wazuh-qa/blob/943b9b03f61959dbf9a01189bace682b73c5bd1f/deployability/README.MD?plain=1#L270

File: deployability/README.MD (hunk 23 above)
Comment (mhamra) [reply deployability/README.MD]: Fixed
image

File: deployability/README.MD (hunk 24 above)
Comment (mhamra) [reply deployability/README.MD]: Fixed
image

File: deployability/README.MD (hunk 25 above)
Comment (mhamra) [reply deployability/README.MD]: fixed in 
[draw-plot.zip](https://github.com/user-attachments/files/15792115/draw-plot.zip)

File: deployability/modules/allocation/README.MD (hunk 26 above)
Comment (mhamra) [reply deployability/modules/allocation/README.MD]: Fixed
https://github.com/wazuh/wazuh-qa/blob/943b9b03f61959dbf9a01189bace682b73c5bd1f/deployability/modules/allocation/README.MD?plain=1#L72

File: deployability/modules/allocation/README.MD (hunk 28 above)
Comment (mhamra) [reply deployability/modules/allocation/README.MD]: Fixed
image

File: deployability/modules/allocation/README.MD (hunk 27 above)
Comment (mhamra) [reply deployability/modules/allocation/README.MD]: Fixed

https://github.com/wazuh/wazuh-qa/blob/746eb38dfeea6a992b7d4c4512a0dcae27561ea4/deployability/modules/allocation/README.MD?plain=1#L353

File: deployability/modules/allocation/README.MD (hunk 29 above)
Comment (mhamra) [reply deployability/modules/allocation/README.MD]: Fixed
[Allocation.drawio.zip](https://github.com/user-attachments/files/15792190/Allocation.drawio.zip)

File: deployability/modules/jobflow/README.MD (hunk 30 above)
Comment (mhamra) [reply deployability/modules/jobflow/README.MD]: Fixed
https://github.com/wazuh/wazuh-qa/blob/746eb38dfeea6a992b7d4c4512a0dcae27561ea4/deployability/modules/jobflow/README.MD?plain=1#L5

File: deployability/modules/jobflow/README.MD (hunk 31 above)
Comment (mhamra) [reply deployability/modules/jobflow/README.MD]: Fixed
https://github.com/wazuh/wazuh-qa/blob/746eb38dfeea6a992b7d4c4512a0dcae27561ea4/deployability/modules/jobflow/README.MD?plain=1#L32

File: deployability/modules/jobflow/README.MD (hunk 33 above)
Comment (mhamra) [reply deployability/modules/jobflow/README.MD]: Fixed
https://github.com/wazuh/wazuh-qa/blob/746eb38dfeea6a992b7d4c4512a0dcae27561ea4/deployability/modules/jobflow/README.MD?plain=1#L258

I also fixed all other 'jobflow  ' strings with double space.

File: deployability/modules/jobflow/README.MD (hunk 34 above)
Comment (mhamra) [reply deployability/modules/jobflow/README.MD]: Fixed 
https://github.com/wazuh/wazuh-qa/blob/746eb38dfeea6a992b7d4c4512a0dcae27561ea4/deployability/modules/jobflow/README.MD?plain=1#L262

**Comment by mhamra** [comment]
Fixed .gitignore

https://github.com/wazuh/wazuh-qa/blob/746eb38dfeea6a992b7d4c4512a0dcae27561ea4/.gitignore#L9

**Review by rauldpm — CHANGES_REQUESTED** [review]
We are going to take another approach to this PR

1. Revert all commits
2. Change README titles
3. Change the directory module from `workflow_engine` to `jobflow`
4. Change all code that is affected by this directory change
5. Review code and change `Workflow` (as name) to `JobFlow`, `workflow` is a word and has a meaning, take that into account

**Comment by mhamra** [comment]
### UPDATE
I've implemented the requested changes of this [comment]( https://github.com/wazuh/wazuh-qa/pull/5460#pullrequestreview-2113617699) in https://github.com/wazuh/wazuh-qa/pull/5460/commits/5d9d78008add12f871d93b2bc6609e0c59cbc448

**Review by fcaffieri — APPROVED** [review]

**Review by rauldpm — APPROVED** [review]

**Pull request merged**